images/installer/Dockerfile.openstack.ci

```diff
@@ -0,0 +1,26 @@
+# This Dockerfile is a used by CI to test using OpenShift Installer against an OpenStack cloud.
+# It builds an image containing the openshift-install command as well as the openstack cli.
+
+
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
+WORKDIR /go/src/github.com/openshift/installer
+COPY . .
+RUN hack/build.sh
+
+FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
+COPY --from=builder /go/src/github.com/openshift/installer/bin/openshift-install /bin/openshift-install
+
+RUN yum install --setopt=tsflags=nodocs -y \
+    https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm \
```

Comment:
@abhinavdahiya In my local tests, the epel-release and rdo-release rpms were not available in the base image. I had to install them from https links.

Reply:
do you have log from failure...
did it look like https://github.com/openshift/installer/pull/1456#discussion_r268363202

Reply:
It just didnt find the package. So it fails silently, but then the openstackclient is not installed.